mmv1/templates/terraform/examples/artifact_registry_repository_remote_common_repository_with_docker.tf.tmpl

```diff
@@ -0,0 +1,20 @@
+resource "google_artifact_registry_repository" "upstream_repo" {
+  location      = "us-central1"
+  repository_id = "{{index $.Vars "upstream_repository_id"}}"
+  description   = "{{index $.Vars "upstream_repository_desc"}}"
+  format        = "DOCKER"
+}
+
+resource "google_artifact_registry_repository" "{{$.PrimaryResourceId}}" {
+  location      = "us-central1"
+  repository_id = "{{index $.Vars "repository_id"}}"
+  description   = "{{index $.Vars "desc"}}"
+  format        = "DOCKER"
+  mode          = "REMOTE_REPOSITORY"
+  remote_repository_config {
+    description = "pull-through cache of another Artifact Registry repository"
+    common_repository {
+      uri         = upstream_repo.id
```

Comment:
```suggestion
      uri         = google_artifact_registry_repository.upstream_repo.id
```

Reply:
Do you want to refer to the resource `google_artifact_registry_repository.upstream_repo` here?

Reply:
🤦 fixed. ty